Graph - Ranking Changes.do

```diff
@@ -49,5 +49,33 @@ foreach player in $current_regulars {
 drop if ratingorder == ratingorder[_n-1]
 drop foo
 gen foo = -1 * (_N - _n)
-line $current_regulars foo in -`changes_shown'/L, lwidth(3 ..) yscale(reverse) ylabel(1/15, angle(0)) xlabel(-`changes_shown'(10)0) title("CRAPPR Rankings", span) subtitle("Over `changes_shown' Most Recent Ranking Changes", span) xtitle("") legend(col(1) pos(3) symxsize(4) rowgap(3) region(style(none) margin(0 0 0 0)) bmargin(1 0 .25 .25)) xsize(6.5) ysize(4.45) graphregion(color(white) margin(0 0 b+1 t+1))
+
+#d;
+twoway
+	(line $current_regulars foo in -`changes_shown'/L),
+		lwidth(3 ..) 
```

Comment:
```suggestion
	line $current_regulars foo in -`changes_shown'/L,
		lwidth(3 ..) 
```

Code doesn't run as-is. Since there is only one plot, the plot options and the twoway options are all after the comma on this line. Wrapping this line in parentheses indicates to Stata where the plot command (and thus the plot options) end and then the comma would indicate where the twoway options begin. This is useful when there are multiple plots in one twoway command, so that you can specify separate plot options for each plot (e.g., `lwidth()`) and then have overall twoway options.

To illustrate this point, the same result could be achieved with:
```
	(line $current_regulars foo in -`changes_shown'/L, lwidth(3 ..)),
```

Reply:
I see now. Thanks for the explanation; it was helpful. I committed your suggested changes.